resources/stylesheets/inviwo.qss

```diff
@@ -867,20 +868,57 @@ QWidget#CompositeWidget QToolButton#addListItemButton:disabled {
     image: url(:svgicons/listitem-add-disabled.svg);
 }
 
+/************************ 
+QTableView
+************************/
+QTableView QTableCornerButton::section {
+    border: none;
+    border-bottom: 0.125em solid #4a4a4f; 
+    border-right: 0.125em solid #4a4a4f; 
+    background: #2a2a2e;
+}
+QTableView QHeaderView::down-arrow {
+    image: url(:/svgicons/arrow-down-enabled.svg);
+    subcontrol-position: center left;
+    margin-left: 0.2em;
+}
+QTableView QHeaderView::up-arrow {
+    image: url(:/svgicons/arrow-up-enabled.svg);
+    subcontrol-position: center left;
+    padding-left: 0.2em;
+}
+QTableView, QTableWidget {
+    border: none;
+    color: #999999;
```

Comment:
Can selection color be exposed in the processor? Then it would be possible to make it consistent with other plots

Reply:
Quite tricky